packages/code_gen_tester/lib/src/tester.dart

```diff
@@ -71,7 +71,7 @@ class _SourceGenTesterImpl implements SourceGenTester {
       [BuildStep? buildStep]) async {
     final generated =
         await generator.generate(library, buildStep ?? _BuildStepImpl());
-    final output = formatter.format(generated ?? '');
```

Comment:
`generated` is a `String?`, so this produces an analyzer error?

See https://pub.dev/documentation/source_gen/latest/source_gen/Generator/generate.html